src/App.tsx

```diff
@@ -8,8 +8,7 @@ import { TodoList } from './components/TodoList/TodoList';
 import { useClearCompleted } from './hooks/useClearCompleted';
 
 const AppContent: React.FC = () => {
-  const { todos, filteredTodos, error, setFilter, setError, filter } =
-    useTodos();
+  const { todos, filteredTodos, error, filter } = useTodos();
```

Comment:
you don't use filter and filteredTodos you just pass it as props to other components

Reply:
if I change it code stop working